### DISCUSSION TITLE
Activate convex hull rendering in python

### DISCUSSION THREAD

**Discussion by rohit-kumar-j** [google-deepmind/mujoco#336] [Asking for Help] [answered]
How do we activate the convex hull rendering in python?

refer: [`mujoco-python-viewer`](https://github.com/rohanpsingh/mujoco-python-viewer/blob/1a6180b2ba8b798db16b21c01f9717ed33df458f/mujoco_viewer/mujoco_viewer.py#L118)

```python

# toggle joint rendering
mujoco.mjtVisFlag.mjVIS_JOINT = 1 # or 0

# doesn't work
mujoco.mjtVisFlag.mjVIS_CONVEXHULL = 1 # or 0

```

**Comment by nimrod-gileadi** [comment, marked as answer]
`mujoco.mjtVisFlag.mjVIS_CONVEXHULL` is an index into the `flags` array in `MjvOption`, and you shouldn't try to assign to it.

In the line of code you pointed at, you would do something like:

```
self.vopt.flags[mujoco.mjtVisFlag.mjVIS_CONVEXHULL] = 1
```

Is that what you did and it didn't work?

**Reply by rohit-kumar-j** [reply]
No, I had tried:

```xml
mujoco.mjtVisFlag.mjVIS_CONVEXHULL = 1
```
